sdk/src/Services/S3/Generated/Model/DeleteBucketInventoryConfigurationRequest.cs

```diff
@@ -0,0 +1,147 @@
+    /// 
+    /// </para>
+    ///  </li> <li> 
+    /// <para>
+    ///  <a href="https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketInventoryConfiguration.html">PutBucketInventoryConfiguration</a>
+    /// 
+    /// </para>
+    ///  </li> <li> 
+    /// <para>
+    ///  <a href="https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListBucketInventoryConfigurations.html">ListBucketInventoryConfigurations</a>
+    /// 
+    /// </para>
+    ///  </li> </ul> <important> 
+    /// <para>
+    /// You must URL encode any signed header values that contain spaces. For example, if
+    /// your header value is <c>my file.txt</c>, containing two spaces after <c>my</c>, you
+    /// must URL encode this value to <c>my%20%20file.txt</c>.
+    /// </para>
+    ///  </important>
+    /// </summary>
+    public partial class DeleteBucketInventoryConfigurationRequest : AmazonWebServiceRequest
+    {
+        private string _bucketName;
+        private string _expectedBucketOwner;
+        private string _inventoryId;
+
+        /// <summary>
+        /// Gets and sets the property BucketName. 
+        /// <para>
+        /// The name of the bucket containing the inventory configuration to delete.
+        /// </para>
+        /// </summary>
+        [AWSProperty(Required=true)]
+        public string BucketName
+        {
+            get { return this._bucketName; }
+            set { this._bucketName = value; }
+        }
+
+        // Check to see if BucketName property is set
+        internal bool IsSetBucketName()
+        {
+            return this._bucketName != null;
+        }
+
+        /// <summary>
+        /// Gets and sets the property ExpectedBucketOwner. 
+        /// <para>
+        /// The account ID of the expected bucket owner. If the account ID that you provide does
+        /// not match the actual owner of the bucket, the request fails with the HTTP status code
+        /// <c>403 Forbidden</c> (access denied).
+        /// </para>
+        /// </summary>
+        public string ExpectedBucketOwner
+        {
+            get { return this._expectedBucketOwner; }
+            set { this._expectedBucketOwner = value; }
+        }
+
+        // Check to see if ExpectedBucketOwner property is set
+        internal bool IsSetExpectedBucketOwner()
+        {
+            return !String.IsNullOrEmpty(this._expectedBucketOwner);
+        }
+
+        /// <summary>
+        /// Gets and sets the property InventoryId. 
+        /// <para>
+        /// The ID used to identify the inventory configuration.
+        /// </para>
+        /// </summary>
+        [AWSProperty(Required=true)]
+        public string InventoryId
+        {
+            get { return this._inventoryId; }
+            set { this._inventoryId = value; }
+        }
+
+        // Check to see if InventoryId property is set
+        internal bool IsSetInventoryId()
+        {
+            return this._inventoryId != null;
```

Comment:
In V3 this seems to be `string.IsNullOrEmpty` instead: https://github.com/aws/aws-sdk-net/blob/ca8952cf30ef8bcfb0f08fd90fa2fa78957f1416/sdk/src/Services/S3/Custom/Model/DeleteBucketInventoryConfigurationRequest.cs#L61-L64